### ISSUE TITLE
Product fields with in multiples of 1,000 are reduced to a single value

### ISSUE THREAD

**Issue by klhall1987** [wpninjas/ninja-forms#2528]
Product fields that are multiples of 1,000 are reduced to a single digit value. For example 1,000 is shown as 1 in submissions. 10,000 is shown as 10 in submission so on and so forth. 

KR said the issue maybe coming from one of the following lines. 
includes/fields/products.php line 60, 72, and 109

**Comment by andreasupftw** [comment]
This is probably an issue deriving from i18n or rather lack thereof.
It's 1 AM here, so I can't really dive into it. Just wanted to chime in, or something...

**Comment by andrewfleming** [comment]
This could also be a result of [PHPs string conversion to numbers](http://php.net/manual/en/language.types.string.php#language.types.string.conversion), and the quantity field formatting numbers with a comma thousand separator. 

I'm assuming the quantity is saved in the DB as a string, so when it is used in a mathematical equation PHP converts the string to a number following the rules in the link above and everything after the thousand separator is disregarded. 

[Lines 60 -66 of /includes/fields/products.php](https://github.com/wpninjas/ninja-forms/blob/97377f16fe0a552cb0964a0ce09bc1837d35300c/includes/Fields/Product.php#L60-L66) are an examples of this. `floatval` disregards whatever is after the comma thousand seperator too.

The solution I went with when I ran into a similar issue to this was to filter values before they were saved to the DB with `filter_var( $number, FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION );`
